tests/ut/ir/parser/test_remote_store.py

```
@@ -163,7 +164,7 @@ def kernel(
 
 def test_remote_store_rejects_unknown_subop():
     """``pld.tile.<other>`` is rejected at 3-segment dispatch."""
-    with pytest.raises(Exception, match="pld.tile"):
+    with pytest.raises(InvalidOperationError, match="pld.tile"):
```

Ruff (0.16.0)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)